src/pages/Signin/index.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import axios from 'axios';
```

Comment:
Use axios from the helpers file instead

Reply:
@adesege the axios file in helpers works specifically with the access token. At this point, I am using axios because the token has not been yet received.